news/3876.bugfix

```diff
@@ -0,0 +1,2 @@
+Change adapter to @adapts decorator.
```

Comment:
Like on the other PR, `adapts to @adapter` 😄

Reply:
brain's batch processing errors :D